R/util-checkSpec.R

```diff
@@ -93,10 +104,21 @@ CheckSpec <- function(lData, lSpec) {
     }
   }
   if (length(missingCols) > 0) {
-    cli::cli_alert_danger("Not all specified columns in the spec are present in the data, missing columns are: {missingCols}")
+    LogMessage(
+      level = "warn",
+      message = "Not all specified columns in the spec are present in the data, missing columns are: {missingCols}"
+    )
   } else if (length(allCols) > 0) {
-    cli::cli_alert("All {length(allCols)} specified column{?s} in the spec are present in the data: {allCols}")
```

Comment:
@lauramaxwell specifically this `{?s}` was causing the trouble, otherwise I think everything else is ready for review again